src/sailing_robot/scripts/wind_direction_average

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/python
+
+import math
+import collections
+import rospy
+from std_msgs.msg import Float32, Float64
+
+
+
+def angle_average(ang_list):
+    return math.atan2(sum([ math.sin(x*math.pi/180) for x in ang_list]),
+                      sum([ math.cos(x*math.pi/180) for x in ang_list]))*180/math.pi % 360
```

Comment:
It needn't be in a class, but yeah, stick it in the navigation module along with the other angle calculation functions.